releases/gke_travis-staging-1_us-central1-a_gce-staging-1/contour.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: contour
+  namespace: gce-staging-1
+  labels:
+    chart: contour
+spec:
+  releaseName: contour
+  chart:
+    path: charts/contour
+    git: git@github.com:travis-ci/kubernetes-config.git
```

Comment:
We could easily refer to this chart directly, not need to copy the Chart to our code base.

Something like this should do it:
```
chart:
    repository: https://charts.rimusz.net
    name: contour
    version: 0.3.0
```

Reply:
I had to add externalTrafficPolicy, which is not present in original chart

Reply:
Interesting, I didn't figure that out from the commits. 

Maybe this should be visible, like 1 commit 'Import contour Chart' with the unmodified chart and then a commit that does the modification?


Even better would be to fork upstream, make the change, propose the change in a PR upstream and in the mean time refer to the fork in the HelmRelease instead.

Reply:
Also we should up the chart version to make it clear there is a change.